cypress/integration/MultiSelect/filtering/index.js

```diff
@@ -0,0 +1,24 @@
+import '../common'
+import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps'
+
+Given('a filterable MultiSelect with options is rendered', () => {
+    cy.visitStory('MultiSelect', 'With filter field')
+})
+
+When('the user enters a filter string', () => {
+    cy.focused().type('one')
+})
+
+When("the user enters a filter string that doesn't match any options", () => {
+    cy.focused().type('does not exist')
+})
+
+Then('the matching options are displayed', () => {
+    cy.contains('option one')
```

Comment:
To make this more clear when reading the code, you could add `.should('exist')`.
Then it'll show up as an expectation in the cypress ui as well

Reply:
Ah I didn't know that. Changed 👍